packages/sado-connect/src/components/SadoConnectKit.tsx

```diff
@@ -11,6 +11,15 @@ export interface SadoConnectKitProp {
   onViewWallet?: () => void;
 }
 
+/**
+ * Parent React component for SadoConnectKit, in the form of a button.
+ *
+ * @component
+ * @param {string} [props.customLabel="Connect wallet"] - Custom label for the button.
+ * @param {string} [props.customStyle] - Custom CSS style for the button.
+ * @param {Function} [props.onViewWallet] - Callback function to handle viewing wallet.
```

Comment:
should mention optional too?

Reply:
https://jsdoc.app/tags-param.html#optional-parameters-and-default-values

I think it's already correct

Reply:
i mean in the description

Reply:
I think that would be redundant